models/option.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const OptionSchema = new mongoose.Schema({
+    option: {
+        type: String,
+    }
+})
+
+const Option = mongoose.model('Option', OptionSchema);
```

Comment:
We don't want an options model. What we want it to add the model to the question schema and have just a question model.

Reply:
The options schema has two fields, answer and correct. The id will be added for you.
```
Id: mongoose object id
Answer: string
Correct: boolean
```